protolib/traverse.py

```diff
@@ -114,7 +130,8 @@ def scan(root):
         for protocol in get_valid_protocols(root)
         if protocol['status'] != 'empty' and
         # Skipping root dir
-        protocol['slug'] != '.' and not protocol['flags']['ignore']
+        protocol['slug'] != '.' and not protocol['flags']['ignore'] and
+        protocol['flags']['embedded-app'] is False
```

Comment:
This line needs to be removed, or else 'embedded-app' protocols won't be parsed and added to the JSON output. I didn't see any errors or anything when I removed this line and ran this on my own comp, did you?

Reply:
It was giving me an error for one of the tests bc it would throw an error when there wasn't a protocol in the directory.